src/views/homepage/components/product-nav/product-nav.module.css

```diff
@@ -44,7 +44,7 @@
   display: grid;
   grid-template-columns: repeat(2, 1fr);
   gap: var(--homepage-grid-gap);
-  border-radius: 14px;
+  border-radius: 6px;
```

Comment:
💭 A future improvement here would be to use a css variable. Not necessary for this PR, just thinking outloud.